frontend/tests/playwright/testing-doc.md

```diff
@@ -0,0 +1,240 @@
+# Playwright Testing Documentation
+
+This document contains extra information for writing Playwright tests within the context of our application.
+
+-  [Mocking API Requests](#mocking-api-requests)
+   -  [Mock Handlers](#mock-handlers)
+   -  [Static Handlers](#static-handlers)
+      -  [Practical Usage: Developing Components](#practical-usage-developing-components)
+      -  [Practical Usage: Global Test Mock](#practical-usage-global-test-mock)
+   -  [Dynamic Handlers](#dynamic-handlers)
+-  [Debugging Playwright Tests](#debugging-playwright-tests)
+   -  [`pnpm playwright:debug`](#pnpm-playwrightdebug)
+      -  [`--debug` flag](#--debug-flag)
```

Comment:
```suggestion
      -  [`--debug` flag](#debug-flag)
```

Fixes navigation in to this header

Reply:
I was able to test this by inputting the markdown into an editor

https://stackedit.io/

Reply:
Hmm it seems this maybe an issue with `stackedit.io`. 

The links as work for me here https://github.com/iFixit/react-commerce/blob/update-test-docs/frontend/tests/playwright/testing-doc.md and in my local VSCode Previewer.

Reply:
TIL